tests/app/main/views/test_suppliers.py

```diff
@@ -1068,6 +1091,9 @@ def test_should_show_agreement_details_on_page(self, s3, data_api_client):
         assert len(document.xpath('//li[contains(text(), "DG9 0QG")]')) == 1
         # Company number - linked
         assert len(document.xpath('//a[@href="https://beta.companieshouse.gov.uk/company/1234456"][contains(text(), "1234456")]')) == 1  # noqa
+        # Lots
+        assert len(document.xpath('//li[contains(text(), "Lettuce & cucumber")]')) == 1
```

Comment:
If there's a simple way to test the displayed order of lots it might be good to test that too, but not if it's a complete headache.